tools/gen-attributions/ATTRIBUTIONS.md

```diff
@@ -0,0 +1,1251 @@
+# Open Source Software Attribution
+
+[comment]: <> (File generated by gen-attributions. DO NOT EDIT.)
+
+The AWS Controllers for Kubernetes (ACK) source code is licensed under the
+Apache License, version 2.0. A copy of this license is available in the
+[LICENSE](LICENSE) file in the root source code directory and is included,
+along with this document, in any images containing AWS service controller
+binaries.
+
+## Package dependencies
+
+ACK controllers depends on a number of Open Source Go packages. Direct 
+dependencies are listed in the ./go.mod file.
+Those direct package dependencies have some dependencies of their own (known as
+"transitive dependencies")
+
+In this part of the Attribution document, we list our dependent packages and
+include an indication of the Open Source License under which that package is
+distributed. For any package *NOT* distributed under the terms of the Apache
+License version 2.0, we include the full text of the package's License below.
```

Comment:
Below, I see that the spf13/cobra dependency actually *DOES* include the full text of the Apache 2.0 license... is that intended?

Reply:
Initially, i wanted to add that feature but completely forgot. Sending a commit for that very soon.

Reply:
Addressed in 91d6de1363d489acc0d22edc12e9765ebcccc93f, this reduced the size of the ATTRIBUTIONS.md by 900 lines... foewww